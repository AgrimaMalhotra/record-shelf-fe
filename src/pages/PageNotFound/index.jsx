import React from 'react';
import { Header, PageNotFoundMessage } from '../../components';

function PageNotFound() {
  return (
    <>
      <Header />
      <PageNotFoundMessage />
    </>
  );
}

export default PageNotFound;
