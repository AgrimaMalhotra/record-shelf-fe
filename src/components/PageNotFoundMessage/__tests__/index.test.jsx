import { render } from '@testing-library/react';
import * as React from 'react';
import PageNotFound from '..';

describe('Page Not Found', () => {
  it('should render correctly ', () => {
    const { container } = render(<PageNotFound />);
    expect(container).toMatchSnapshot();
  });
});
