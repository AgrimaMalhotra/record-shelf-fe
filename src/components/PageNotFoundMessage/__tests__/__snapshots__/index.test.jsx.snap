// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Not Found should render correctly  1`] = `
<div>
  <div
    class="page-not-found-container"
  >
    <h3>
      404 Error. Page not found
    </h3>
  </div>
</div>
`;
